spec/data/alias_dup_keys.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This file needs to be added to the rakefile under the yamllint_exclude_paths test.